packages/moonstone/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ The following is a curated list of changes in the Enact moonstone module, newest
 - `moonstone/Scroller`, `moonstone/VirtualList.VirtualGridList`, and `moonstone/VirtualList.VirtualList` to show overscroll effects only by wheel input
 - `moonstone/Icon` and `moonstone/IconButton` to require `children`
 
+### Fixed
+
+- `moonstone/Scrollable` to calculate and `scrollTo()` when updated by resizing
```

Comment:
Please use `moonstone/Scroller`, `moonstone/VirtualList.VirtualGridList`, and `moonstone/VirtualList.VirtualList` instead of  `moonstone/Scrollable` because app developers could not know what `moonstone/Scrollable` component is.

Reply:
I'll keep that in mind.